packages/apps-config/src/api/spec/manta.ts

```diff
@@ -11,7 +11,21 @@ const definitions: OverrideBundleDefinition = {
     {
       // on all versions
       minmax: [0, undefined],
-      types: {}
+      types: {
+        Address: 'AccountId',
+        LookupSource: 'AccountId',
+        CurrencyId: {
+          "_enum": [
+            "Native",
+            "DOT",
+            "KSM",
+            "BTC"
```

Comment:
For these, just run `yarn lint` locally and it will be auto-fixed.

Reply:
Thanks! Fixed.